### PR TITLE
Finalize standard

### PR DESCRIPTION
**Pull request by ralphtheninja**
I've tried to make each commit be about the same type of error.

Closes https://github.com/airtap/airtap/issues/87 and https://github.com/airtap/airtap/issues/42

Will squash.

### CODE REVIEW

File: lib/flatten-browserlist.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ function flatten (request, all_browsers) {
         // JS will forget about the .0 when reading any float that can
         // be represented as an integer from yaml, so let's try to
         // match a version of that form as a fallback
-        return browser.version == version || browser.version == version + '.0'
+        return browser.version === version || browser.version === version + '.0'
```
Comment (vweevers) [inline lib/flatten-browserlist.js]: Both `browser.version` and `version` are strings here?

Comment (ralphtheninja) [reply lib/flatten-browserlist.js]: Yes, it seems like it.

File: frameworks/render-stacktrace.js [hunk 2]
```diff
@@ -15,29 +15,29 @@ function getCode (sources, frame) {
...
     return code
   }
 }
 
-function hashByFile (source_map) {
-  var sources = source_map.sources
-  var sources_by_file = {}
+function hashByFile (sourceMap) {
+  var sources = sourceMap.sources
+  var result = {}
```
Comment (vweevers) [inline frameworks/render-stacktrace.js]: I personally prefer `sourcesByFile`, to match the variable name used below.

File: frameworks/zuul.js [hunk 3]
```diff
@@ -198,11 +197,11 @@ ZuulReporter.prototype.done = function (err) {
...
   // add coverage tab content
   if (window.__coverage__) {
-    var coverage_tab = document.getElementById('code-coverage-tab')
-    coverage_tab.innerHTML = '<iframe frameborder="0" src="/__zuul/coverage"></iframe>'
+    var coverageTab = document.getElementById('code-coverage-tab')
+    coverageTab.innerHTML = '<iframe frameborder="0" src="/__zuul/coverage"></iframe>'
   }
 
-  post_message({
+  postMessage({
```
Comment (vweevers) [inline frameworks/zuul.js]: Maybe rename to avoid confusion with `window.postMessage`

Comment (ralphtheninja) [reply frameworks/zuul.js]: `bufferMessage`?

File: lib/sauce-browser.js [hunk 4]
```diff
@@ -140,7 +140,7 @@ SauceBrowser.prototype.start = function () {
         self.emit('start', reporter)
 
         var timeout = false
-        var get_timeout = setTimeout(function () {
+        var timer = setTimeout(function () {
```
Comment (vweevers) [inline lib/sauce-browser.js]: Nice. This was bugging me :)

Comment (ralphtheninja) [reply lib/sauce-browser.js]: Hehe same :) I really wanted to use `timeout` for the handle.

File: frameworks/zuul.js (hunk 3 above)
Comment (vweevers) [reply frameworks/zuul.js]: That'd work. Could also replace these calls with `messageBus.push`. Take your pick :)

File: lib/setup.js [hunk 5]
```diff
@@ -17,8 +17,10 @@ function setupTestServer (opt, cb) {
     setup()
   }
 
-  function setup (_supportServer) {
-    supportServer = _supportServer
+  function setup (err, server) {
+    if (err) return cb(err)
+
+    supportServer = server
```
Comment (ralphtheninja) [inline lib/setup.js]: I'm wondering about this `supportServer` stuff and if we can get rid of it.

Comment (ralphtheninja) [reply lib/setup.js]: It would mean we can throw away `user-server.js` completely.

Comment (vweevers) [reply lib/setup.js]: Also had that thought. I guess there are use cases for it, but the current implementation doesn't seem very solid. We could always bring it back if people need it (and then rethink the approach).

Comment (ralphtheninja) [reply lib/setup.js]: I don't even know what it does. Is it for integrating with external systems?

Comment (vweevers) [reply lib/setup.js]: See https://github.com/airtap/airtap/blob/master/doc/airtap.yml.md#server-optional. You can use this feature to for example run a websocket server and run integration tests with airtap. Airtap will setup and teardown the server.

Comment (vweevers) [reply lib/setup.js]: The alternative is `start-server.js && airtap test.js && kill-server.js`.

**Comment by feross** [comment]
Nice! ✨

**Pull request merged**